media-radio/KochMorse/KochMorse-3.4.0-r1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Morse-code tutor using the Koch method"
+HOMEPAGE="https://github.com/hmatuschek/kochmorse"
+SRC_URI="https://github.com/hmatuschek/kochmorse/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtmultimedia
+	dev-qt/qtwidgets:5"
+DEPEND="${RDEPEND}
+	dev-qt/linguist-tools"
```

Comment:
1) missing slot
2) in EAPI-7, this is in `BDEPEND` instead

Reply:
I'll add the slot. As for `BDEPEND`, the build currently uses EAPI 6, and I don't feel confident changing that given that I cannot build myself.